commands/service_execute.go

```diff
@@ -153,10 +153,17 @@ func readJSONFile(path string) (string, error) {
 	if path == "" {
 		return "{}", nil
 	}
+
 	data, err := ioutil.ReadFile(path)
 	if err != nil {
 		return "", err
 	}
+
+	var raw json.RawMessage
```

Comment:
Do we need this test here ? The test is already done by the API
With this test
```
invalid character '"' after object key:value pair
```
Without the test
```
rpc error: code = Unknown desc = invalid character '"' after object key:value pair
```

I would recommend that if it's an error that the API catch we should let the API catch this. What do you think ?

Reply:
We don't but If we network comes into play, callers of API should do as much validation as possible to save the network overhead. So if we can easily validate it here and get user faster response (which increase user experience) why not to?